pkg/importer/vddk-datasource.go

```diff
@@ -0,0 +1,262 @@
+/*
+Copyright 2020 The CDI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package importer
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"k8s.io/klog"
+)
+
+const (
+	cryptLibrary          = "/usr/lib64/libcrypt.so.1"
+	destinationFile       = "/data/disk.raw"
```

Comment:
I believe the output file needs to be called `disk.img` rather than `disk.raw` for Kubevirt to be able to boot the VM.